ui/components/MesheryApplications.js

```diff
@@ -515,3 +563,284 @@ const mapStateToProps = (state) => {
 
 // @ts-ignore
 export default withStyles(styles)(connect(mapStateToProps, mapDispatchToProps)(withSnackbar(MesheryApplications)));
+
```

Comment:
Is this set of code being repeated across patterns, apps, and filters? Perhaps, this should be centralized.

Reply:
The configuration is there including the RJSF and Service Form in the MesheryInterface but It needs to be added separately in each because they all use a different set of data for eg: application_file, pattern_file(many more). assigned differently in both according to the config, only few like PatternName which is inherited commonly across all